css/styles.css

```diff
@@ -415,7 +424,20 @@ table,
 
 @media only screen and (min-width: 768px) {
   .thumbnail {
-    width: 180px;
+    max-width: 180px;
+  }
+
+  .col-35 {
+    flex-basis: 35%;
+  }
+
+  .col-65 {
+    flex-basis: 65%;
+  }
```

Comment:
You should do this media query in the same file where the class had styling originally applied to it: `layout.css`